src/rest-server/src/utils/manager/storage/crudK8sSecret.js

```diff
@@ -323,6 +328,10 @@ async function patchStorageServer(op, key, value, config) {
  * @return {Promise<Object>} A promise to patch result.
  */
 async function patchStorageConfig(op, key, value, config) {
```

Comment:
Hi Dian, can you show some examples here for this API. I am not sure weather it should return `400 Bad Request` or should return `401 Forbidden` here.
From your error message, it seems caused by wrong request body, not authentication header. So using `BadRequest` error is better?

Reply:
According to https://stackoverflow.com/questions/29912305/response-code-400-or-403-for-post-restful-apis seems 403 Forbidden is better here